lib/repositories/mempool_api_repository.dart

```diff
@@ -0,0 +1,26 @@
+import 'dart:convert';
+
+import 'package:danawallet/constants.dart';
+import 'package:http/http.dart' as http;
+import 'package:danawallet/data/models/mempool_api_fees_recommended_model.dart';
+
+class MempoolApiRepository {
```

Comment:
Instead of calling this MempoolApiRepository, I think it makes more sense to call it 'RecommendedFeesRepository. We use mempool right now but maybe we'll change it in the future.

Reply:
Yeah makes sense, will do asap